pages/docs/extensions/google-calendar.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 layout: 'ExtensionPageLayout'
 description: 'The Google Calendar to Roam Integration allows users to import the list of events on a given day into their daily notes page.'
-contributors: Eric Anderson 💵, Joe Ocampo 💵, George Corple 💵, Marco Brambilla 💵💻, David Schulman 💵
+contributors: Eric Anderson 💵, Joe Ocampo 💵, George Corple 💵, Marco Brambilla 💵💻, David Schulman, 💵 Monty Kosma 💵
```

Comment:
```suggestion
contributors: Eric Anderson 💵, Joe Ocampo 💵, George Corple 💵, Marco Brambilla 💵, David Schulman 💵, Monty Kosma 💻
```

Reply:
Also feel free to add yourself here so that this auto resolves to a link!

https://github.com/dvargas92495/roam-js-extensions/blob/master/components/ExtensionPageLayout.tsx#L61